addons/hr_timesheet/models/hr_timesheet.py

```diff
@@ -74,7 +74,7 @@ def _compute_display_name(self):
         super(AccountAnalyticLine, self - analytic_line_with_project)._compute_display_name()
         for analytic_line in analytic_line_with_project:
             if analytic_line.task_id:
-                analytic_line.display_name = f"{analytic_line.project_id.display_name} - {analytic_line.task_id.display_name}"
+                analytic_line.display_name = f"{analytic_line.project_id.sudo().display_name} - {analytic_line.task_id.sudo().display_name}"
```

Comment:
```suggestion
                analytic_line.display_name = f"{analytic_line.project_id.sudo().display_name} - {analytic_line.task_id.display_name}"
```

does not seem needed to add the second sudo. The user should be able to access to the task.